src/commands/moderation/welcome.ts

```diff
@@ -79,7 +79,7 @@ const HELP_EMBED = makeEmbed({
 export const welcome: CommandDefinition = {
     name: 'welcome',
     description: 'Sends the welcome',
-    requiredPermissions: ['BanMembers'],
+    requirements: { permissions: ['BanMembers'] },
```

Comment:
Not a comment on the PR, but @benw202 : Does this make sense, should we take the opportunity to change this to ADMIN & MODERATOR roles?